docs/config.rst

```diff
@@ -326,13 +329,16 @@ Themes
 If you're using themes from the WordPress.org repository you can add them in a list using the themes slug.
 These will be downloaded for you. The last theme in the list will be the theme that is activated for your site.
 
+Alternatively, if you want to install a plugin from a Git repository you can use a URL to a zip file of your theme. e.g. ``https://github.com/humanmade/S3-Uploads/archive/master.zip``
```

Comment:
Did you want to use `theme` here instead of `plugin` here? i.e `Alternatively, if you want to install a theme...`

Reply:
I sure do! Good catch. Thanks!